### DISCUSSION TITLE
Where to get an image tagged for tensorrt ?

### DISCUSSION THREAD

**Discussion by The-NinthWave** [blakeblackshear/frigate#9226] [Ask A Question] [answered]
According to this excerpt from installation docs, I should get an image with a tag for tensorrt but can't find any.
Where are they ?
![image](https://github.com/blakeblackshear/frigate/assets/53533635/273136e3-ace7-4180-9f54-2f12901a8174)

**Comment by NickM-27** [comment]
`ghcr.io/blakeblackshear/frigate:stable-tensorrt` for example. They are all listed on the installation docs

**Reply by The-NinthWave** [reply]
Thanks for the link.

But let's say one wants to choose a special tag other than "stable"

I found this "page" using google: 
![image](https://github.com/blakeblackshear/frigate/assets/53533635/f098b474-37a6-40e1-9b70-b4b456767baf)


But from the main Github page, I can't find the path to get there
![image](https://github.com/blakeblackshear/frigate/assets/53533635/e2b2aa30-292c-401a-b7b1-544565c0ceea)

**Reply by NickM-27** [reply, marked as answer]
![Screen Shot 2024-01-06 at 11 01 13 AM](https://github.com/blakeblackshear/frigate/assets/14866235/211faf8d-507f-4379-a407-c41f63cdc606)

**Reply by The-NinthWave** [reply]
Jeez... I assure you I looked everywhere but apparently I did not scroll down enough.

I guess that's what happens at >50 when presbyopia makes one zoom the screen at 130% even with progressive glasses.

THANKS
